appinventor/components/src/com/google/appinventor/components/runtime/util/RuntimeErrorAlert.java

```diff
@@ -19,22 +20,33 @@
  * @author halabelson@google.com (Hal Abelson)
  */
 public final class RuntimeErrorAlert {
+
+  private static final String TAG = "RuntimeErrorAlert";
+
   public static void alert(final Object context,
```

Comment:
We should be careful about this change here. I would suggest having an overloaded method that retains the original function signature and have that implementation call this enhanced version. Otherwise, if there are any extensions that rely on this version of the signature they will break on an upgrade of App Inventor.

Reply:
Done